tema1/script3.js

```diff
@@ -0,0 +1,29 @@
+console.log("EXERCITIUL 3");
+
+var a = "Luni";
+Luni = 0;
+console.log(`${a}: ${Luni}`);
+
+var a = "Marti";
```

Comment:
Ceea ce ai facut tu aici se numeste redeclararea variabilei a (deoarece folosesti iar un declarator cum ar fi var(de preferat let)). Aceasta practica este gresita deoarece e ca si cum in loc sa schimbi continutul unui dulap, schimbi dulapul cu totul. Ar fi fost suficient sa atribui direct
```js
a = "Marti";
```